Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,260 @@
+                {
+                    item.Children = GetTypeItems(@namespace);
+                }
+                else if (item.Data is Type type)
+                {
+                    item.Children = GetMethodItems(type);
+                }
+                tree.RefreshItem(item);
+            }
+        }
+        private void Refresh()
+        {
+            var rootItems = _assemblies.Select(a => new TreeViewItem(a.GetName().Name, a, GetFullNameItems(a), CacheLoadTypes(a).Any(), a.GetName().Version.ToString()))
+                .Where(tvi => tvi.Children.Any())
+                .ToList();
+            tree.SetItems(rootItems);
+        }
+        private List<TreeViewItem> GetFullNameItems(Assembly assembly)
+        {
+            var clt = CacheLoadTypes(assembly);
+            var data = new List<TreeViewItem>();
+            hashSet.Clear();
+            for (int i = 0; i < clt.Length; i++)
+            {
+                var type = clt[i];
+                if (string.IsNullOrEmpty(type.Namespace)) continue;
+                if (hashSet.Contains(type.Namespace))
+                    continue;
+                hashSet.Add(type.Namespace);
+                var name = type.Name;
+                data.Add(new TreeViewItem(type.Namespace, type.Namespace, null, true));
+            }
+
+            return data;
+        }
+        private List<TreeViewItem> GetTypeItems(string @namespace)
+        {
+            var allTypes = _assemblies.SelectMany(CacheLoadTypes);
+            var types = allTypes
+                .Where(t => t.Namespace == @namespace)
+                .OrderBy(t => t.Name)
+                .ToList();
+
+            return types.Select(t =>
+                {
+                    var mtvi = GetMethodItems(t);
+
+                    var qualifiers = "<color=#569cd6>" 
+                                   + (t.IsAbstract ? "abstract " : string.Empty)
+                                   + (t.IsValueType ? "struct " : "class ")
+                                   + "</color>";
+                    var name = qualifiers + HighlightTypeName(t);
+                    TreeViewItem treeViewItem = new TreeViewItem(name, t, mtvi, mtvi.Count > 0);
+                    return treeViewItem;
+                })
+                .Where(tvi => tvi.Children.Count > 0)
+                .ToList();
+        }
+        private Type[] CacheLoadTypes(Assembly assembly)
+        {
+            if (!_assemblyTypes.ContainsKey(assembly))
+                _assemblyTypes[assembly] = assembly.GetTypes();
+
+            return _assemblyTypes[assembly];
+        }
+        private List<TreeViewItem> GetMethodItems(Type type)
+        {
+            var methods = type.GetMethods(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance | BindingFlags.DeclaredOnly)
+                .Where(m => !m.IsGenericMethod && m.GetMethodBody() != null && !m.MethodImplementationFlags.HasFlag(MethodImplAttributes.InternalCall))
```

Comment:
Maybe break out this check into a static `IsSupportedForJitInspection` method. It'll make it more convenient to tweak the checks later, and also perform it in multiple spots if necessary.